api/v1beta1/azuremanagedcluster_types.go

```diff
@@ -37,6 +37,9 @@ type AzureManagedClusterStatus struct {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels.cluster\\.x-k8s\\.io/cluster-name",description="Cluster to which this AzureManagedCluster belongs"
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=".status.conditions[?(@.type=='Ready')].status"
```

Comment:
I think this still won't work, since the `JSONPath` query refers to `conditions`.